modules/javamail/src/main/scala/emil/javamail/JavaMailEmil.scala

```diff
@@ -29,7 +29,7 @@ final class JavaMailEmil[F[_]: Sync: ContextShift] private (
     new SendImpl[F](blocker)
 
   def access: Access[F, JavaMailConnection] =
-    new AccessImapImpl[F](blocker)
+    new AccessGimapImpl[F](blocker)
```

Comment:
I appreciate a lot that the main api is not changing :). Just wondering how you can use these features, do you cast the `Access` object to the Gimap one?

Reply:
Yes, precisely. I don't want to make this as part of the public interface _just yet_, in case we realize we need to change a few things around.
People who call `asInstanceOf` (including us) need to count with the possibility of API breakage.

Reply:
Yes, that's a very good idea, thank you! 👍🏼